File: 240205/두 개의 동일한 수열/two-equal-series.py
n = int(input())
a = list(map(int, input().split()))
b = list(map(int, input().split()))
cnt = 0

a.sort()
b.sort()

for i in range(n):
    if a[i] != b[i]:
        cnt +=1
        break

if cnt == 0:
    print('Yes')
else:
    print('No')

File: 240812/두 개의 동일한 수열/two-equal-series.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split("\n");
let n = Number(input[0]);
let arrA = input[1].trim().split(" ").map(Number);
let arrB = input[2].trim().split(" ").map(Number);

arrA.sort((prev, cur) => prev - cur);
arrB.sort((prev, cur) => prev - cur);

let satisfied = true;
for (let i = 0; i < n; i++) {
    if (arrA[i] !== arrB[i]) satisfied = false;
}

if (satisfied) {
    console.log("Yes");
} else {
    console.log("No");
}